packages/library-base/src/commonMain/kotlin/io/realm/kotlin/Configuration.kt

```diff
@@ -170,6 +170,9 @@ public interface Configuration {
      */
     public val inMemory: Boolean
 
+    public val assetFile: String?
+    public val assetFileChecksum: String?
```

Comment:
We need documentation for these properties

Reply:
Should we perhaps move them into a dataclass? like `AssetFile(path, sha256checksum)` ? It would keep the number of properties in the configuration down and these are connected anyway? Similar to what we did for the `LogConfiguration`

Reply:
It could make sense to bundle them up to reduce the number of properties, but I assume we should keep the 
```
public fun assetFile(assetFile: String, sha256checkSum: String? = null)
```
as is and not use the `AssetFileConfiguration`-data class as argument!? Then it becomes a bit tricky where to keep the main documentation. Should we use the `assetFile`-method as main entry for explaining the concepts, as this is where users would look in the first place and then just keep the other docs brief and reference this?

Reply:
Yeah, I would keep this method 👍 And yes, I agree that since this is the main entry point, I would keep the bulk of the docs here and just keep it light on the final Configuration class

Reply:
Bundled the properties up in the `AssetFileConfiguration`, but kept `assetFile` and docs here.